src/player.js

```diff
@@ -170,7 +163,7 @@ export default class Player extends FakeEventTarget {
   static getCapabilities(engineType: ?string): Promise<{ [name: string]: any }> {
     Player._logger.debug("Get player capabilities", engineType);
     const promises = [];
```

Comment:
maybe a more meaningful name, like avaiableEngines (though its not exactly that)